src/components/layout.js

```diff
@@ -29,7 +29,7 @@ const Layout = ({ children }) => {
   `);
 
   return (
-    <>
+    <React.Fragment>
```

Comment:
using `<>` breaks linter in VS code. It's better to use `React.Fragment` which is essentially the same thing.

![image](https://user-images.githubusercontent.com/5561498/68046455-38f38980-fcdc-11e9-8047-e7572f9c6689.png)

With React.Fragment

![image](https://user-images.githubusercontent.com/5561498/68046486-4a3c9600-fcdc-11e9-9822-bf90808f8fbb.png)